formidling/src/main/java/no/nav/ung/sak/formidling/innhold/InnvilgelseInnholdBygger.java

```diff
@@ -44,219 +40,152 @@ public class InnvilgelseInnholdBygger implements VedtaksbrevInnholdBygger {
-            throw new IllegalStateException("Fant ingen tilkjente perioder");
-        }
+        return resultat;
 
-        return null;
     }
 
-    private static List<TilkjentPeriodeDto> mapTilTilkjentYtelseDto(LocalDateTimeline<GrunnlagOgTilkjentYtelse> grunnlagOgTilkjentYtelseTidslinje, UngdomsytelseSatsType satsType) {
-        return grunnlagOgTilkjentYtelseTidslinje
-            .filterValue(it -> it.satsType() == satsType)
-            .stream()
-            .map(s -> {
-                var it = s.getValue();
-                return new TilkjentPeriodeDto(
-                    new PeriodeDto(s.getFom(), s.getTom()),
-                    new TilkjentYtelseDto(it.dagsats(), it.grunnbeløpFaktor(), it.grunnbeløp(), it.årsbeløp()));
-            }).toList();
-    }
+    private static SatsEndringHendelseDto lagSatsEndringHendelseDto(BrevfeilHåndterer brevfeilHåndterer, LocalDateSegment<UngdomsytelseSatser> current, LocalDateSegment<UngdomsytelseSatser> previous) {
+        var currentSatser = current.getValue();
+        var previousSatser = previous.getValue();
 
-    private ResultatFlaggDto lagResultatFlaggDto(
-        LocalDateTimeline<GrunnlagOgTilkjentYtelse> grunnlagOgTilkjentYtelseTimeline,
-        VurderAntallDagerResultat vurderAntallDagerResultat,
-        Behandling behandling) {
-
-        var grunnlagOgTilkjentYtelse = grunnlagOgTilkjentYtelseTimeline.stream()
-            .map(LocalDateSegment::getValue)
-            .distinct()
-            .toList();
-
-
-        boolean enDagsats = grunnlagOgTilkjentYtelse.stream().collect(Collectors.groupingBy(GrunnlagOgTilkjentYtelse::dagsatsTilkjentYtelse)).size() == 1;
-        boolean ettGbeløp = grunnlagOgTilkjentYtelse.stream().collect(Collectors.groupingBy(GrunnlagOgTilkjentYtelse::grunnbeløp)).size() == 1;
-
-        var satsTyper = grunnlagOgTilkjentYtelse.stream().map(GrunnlagOgTilkjentYtelse::satsType).collect(Collectors.toSet());
-        boolean lavSats = satsTyper.stream().allMatch(it -> it == UngdomsytelseSatsType.LAV);
-        boolean høySats = satsTyper.stream().allMatch(it -> it == UngdomsytelseSatsType.HØY);
-        boolean varierendeSats = satsTyper.contains(UngdomsytelseSatsType.LAV) && satsTyper.contains(UngdomsytelseSatsType.HØY);
-
-        LocalDate fødselsdato = personopplysningRepository.hentPersonopplysninger(behandling.getId())
-            .getGjeldendeVersjon()
-            .getPersonopplysning(behandling.getAktørId())
-            .getFødselsdato();
-        //TODO er dette mulig?
-        boolean oppnårMaksAlder = vurderAntallDagerResultat.tidslinjeNokDager()
-            .getMaxLocalDate()
-            .isAfter(fødselsdato.plusYears(Sats.HØY.getTomAlder()));
-
-        return new ResultatFlaggDto(
-            enDagsats,
-            ettGbeløp,
-            lavSats,
-            høySats,
-            varierendeSats,
-            oppnårMaksAlder);
-    }
+        var fødselBarn = currentSatser.antallBarn() > previousSatser.antallBarn();
```

Comment:
`currentSatser.antallBarn()` og `previousSatser.antallBarn()` er kalt mange ganger i koden her.
Ville vurdert å definere variabler for `gjeldendeBarn` og `tidligereBarn` f.eks.